client/src/components/Alert/Alert.tsx

```diff
@@ -1,16 +1,17 @@
-import React from "react";
-import { UncontrolledAlert } from "reactstrap";
+import React, { ReactElement } from 'react';
+import { UncontrolledAlert } from 'reactstrap';
 
-interface AlertProps {
-  clearAlert: Function;
-  message: string;
-  color: string;
+interface IAlertProps {
+    clearAlert: any;
+    message: string;
+    color: string;
 }
 
-const Alert: React.FunctionComponent<AlertProps> = ({
+const Alert: React.FunctionComponent<IAlertProps> = ({
   clearAlert,
   message,
-  color
-}) => message ? <UncontrolledAlert color={color} onClick={() => { clearAlert() }}>{message}</UncontrolledAlert> : null
+  color,
+}: {clearAlert: () => void, message: string, color: string}): ReactElement | null => message ?
```

Comment:
maybe && operator will be better?

Reply:
it will be better to make it in container

Reply:
add - 
message &&
    <UncontrolledAlert color={color} onClick={(): void => { cleanAlert(); }}>{message}</UncontrolledAlert>;